scripts/deploy.ts

```diff
@@ -1,85 +0,0 @@
-/**
- * Script to deploy warp routes
- * Accepts 3 arguments:
- *   - private-key : Hex string of private key. Note: deployment requires funds on all chains
- *   - token-config : Path to token config JSON file (see example in ./configs)
- *   - chain-config : (Optional) Path to chain config JSON file (see example in ./configs)
- * Example: yarn ts-node scripts/deploy.ts --private-key $PRIVATE_KEY --token-config ./configs/warp-route-token-config.json
- */
-import { ethers } from 'ethers';
-import fs from 'fs';
-import path from 'path';
-import yargs from 'yargs';
-
-import {
-  chainMetadata,
-  MultiProvider,
-  serializeContracts,
-} from '@hyperlane-xyz/sdk';
-
-import { HypERC20Deployer } from '../src/deploy';
-
-async function deployWarpRoute() {
-  const argv = await yargs
```

Comment:
Can we remove yargs from dependencies?

Reply:
Good idea!

Reply:
Actually it's not there, an oversight when this script was originally written